ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/electra/helpers/BeaconStateAccessorsElectra.java

```diff
@@ -60,9 +60,8 @@ public UInt64 getActivationExitChurnLimit(final BeaconStateElectra state) {
    */
   public UInt64 getActiveBalance(final BeaconState state, final int validatorIndex) {
     final Validator validator = state.getValidators().get(validatorIndex);
-    final UInt64 maxEffectiveBalance = miscHelpers.getMaxEffectiveBalance(validator);
     final UInt64 validatorBalance = state.getBalances().get(validatorIndex).get();
-    return validatorBalance.min(maxEffectiveBalance);
+    return validatorBalance.min(validator.getEffectiveBalance());
```

Comment:
Let's remove this function and move the logic in place on processPendingConsolidation. This will bring us closer to the spec code (`get_active_balance` does not exist anymore).

Reply:
Got it. Makes sense.